woocommerce/woocommerce
# Payment Gateways: Update Airwallex png

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Swap out the existing Airwallex logo for new image in Payment Gateways

<img width="712" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/03a1ec3a-93c0-48b7-80b9-bd205f88456e">



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Start with a fresh install
1. Go to core profiler
1. Choose `UK` as store country
1. Complete core profiler without installing any plugins
1. Go to `WooCommerce > Settings > Advanced > WooCommerce.com` > **turn off** "Show Suggestions" checkbox
1. Go to `WooCommerce -> Home` and choose the `Get paid` task.
2. See the new airwallex logo. For reference, here is the old:

<img width="765" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/e3ac3e2f-1fe3-46cd-8bf1-d78ebfdd5629">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Non functional change: update airwallex image

</details>
